sentera/weather.py

```
@@ -328,18 +324,17 @@ def _merge_to_full_df(weather_variable, weather_interval, response_json, data_df
                 columns=[TIME_COLUMNS[weather_interval], "lat", "long"]
             )
 
-        for f in tqdm.tqdm(asyncio.as_completed(tasks), total=len(tasks)):
+        disable_tqdm = strtobool(os.environ.get("DISABLE_TQDM") or "false")
+        for f in tqdm.tqdm(
+            asyncio.as_completed(tasks), total=len(tasks), disable=disable_tqdm
+        ):
             response, weather_variable, url = await f
-            try:
-                response_json = json.loads(response)
-                if weather_type == WeatherType.SevenDay:
-                    data_df = _combine_seven_day(url, response_json, data_df)
-                else:
-                    data_df = _merge_to_full_df(
-                        weather_variable, weather_interval, response_json, data_df
-                    )
-            except Exception as e:
-                print(response)
-                raise e
```

Oh also, I don't think exceptions should ever have been swallowed.  This was only put here to print more information about the what might have gone wrong before re-raising the exception.  Totally fine if you don't want to print the response anymore, though.

---

It was swallowed here
https://github.com/SenteraLLC/py-sentera-api/pull/16/files#diff-34ec00514564ad0696087035ae2d4bf9L277-L283